File: android/src/main/java/dev/barooni/capacitor/calendar/models/results/CreateEventWithPromptResult.kt
package dev.barooni.capacitor.calendar.models.results

import com.getcapacitor.JSObject
import dev.barooni.capacitor.calendar.models.templates.JSResult

class CreateEventWithPromptResult(
    private val id: Long? = null,
) : JSResult {
    override fun toJSON(): JSObject {
        val result = JSObject()
        result.put("id", id?.toString())
        return result
    }
}

File: ios/Plugin/Models/Results/CreateEventWithPromptResult.swift
import Capacitor

struct CreateEventWithPromptResult: JSResult {
    private let id: String?

    init(id: String?) {
        self.id = id
    }

    func toJSON() -> JSObject {
        var result = JSObject()
        result["id"] = id ?? NSNull()
        return result
    }
}
